packages/python-pulp-cli/python-pulp-cli.spec

```diff
@@ -0,0 +1,63 @@
+# Created by pyp2rpm-3.3.6
+%global pypi_name pulp-cli
+
+Name:           python-%{pypi_name}
+Version:        0.10.1
+Release:        1%{?dist}
+Summary:        Command line interface to talk to pulpcore's REST API
+
+License:        GPLv2+
+URL:            https://github.com/pulp/pulp-cli
+Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python%{python3_pkgversion}-devel
+BuildRequires:  python%{python3_pkgversion}-setuptools
+
+%description
+%{summary}
+
+%package -n     python%{python3_pkgversion}-%{pypi_name}
+Summary:        %{summary}
+%{?python_provide:%python_provide python%{python3_pkgversion}-%{pypi_name}}
+
+Requires:       python%{python3_pkgversion}-PyYAML < 5.5
+Requires:       python%{python3_pkgversion}-PyYAML >= 5.4.1
+Requires:       python%{python3_pkgversion}-click < 9
+Requires:       python%{python3_pkgversion}-click >= 7.1.2
+Requires:       python%{python3_pkgversion}-click-shell < 3
+Requires:       python%{python3_pkgversion}-click-shell >= 2.1
+Requires:       python%{python3_pkgversion}-packaging
+Requires:       python%{python3_pkgversion}-pygments
+Requires:       python%{python3_pkgversion}-requests < 2.26
+Requires:       python%{python3_pkgversion}-requests >= 2.25.1
+Requires:       python%{python3_pkgversion}-setuptools
+Requires:       python%{python3_pkgversion}-toml = 0.10.2
```

Comment:
Why is this pinned on an exact version? Does upstream have a good reason for that? https://github.com/pulp/pulp-cli/commit/d3491ef903a4d166124f546faee326851d29ab47 doesn't really mention it.

Reply:
the lib doesn't offer a stable api (yet) :(

Reply:
I guess that would have been too convenient for the ecosystem.